cypress/locators/drawer/index.js

```diff
@@ -3,6 +3,6 @@ import {
 } from './locators';
 
 // component preview locators
-export const drawerToggle = drawer => cy.iFrame(`#${drawer}`).find(DRAWER_TOGGLE);
-export const drawerSidebar = drawer => cy.iFrame(`#${drawer}`).find('div:nth-child(1)');
+export const drawerToggle = () => cy.get(DRAWER_TOGGLE);
+export const drawerSidebar = drawer => cy.get(`#${drawer}`).find('div:nth-child(1)');
```

Comment:
could we move `#${drawer}` as a separate locator into `locators.js` file ?

Reply:
It's not a permanent locator, so it's not a good idea